src/ui/components/Difference/index.jsx

```diff
@@ -21,13 +22,17 @@ import processDifferences from "../../../utils/processDifferences";
 function Difference() {
   const [toast, setToast] = useState({});
   const [isOpenedPopup, setIsOpenedPopup] = useState(false);
+  const [isComparable, setIsComparable] = useState(false);
+  const [isPaginationClicked, setIsPaginationClicked] = useState("");
```

Comment:
- `P1`
```suggestion
  const [clickedType, setClickedType] = useState({});
```

1. 코드상에서 할당되는 값이 객체로 확인됩니다!
따라서 is로 시작하는 변수명은 부적절하다고 판단되네요!

2. 해당 state가 할당되는 type을 초깃값으로 설정해주면 해당 타입이 어떤 type을 값으로 갖게 되는지 유추 할 수 있게 되서 가독성에 도움이 될 것 같아요!
지금같은 상황에는 `빈 객체`가 초깃값으로 들어가면 좋을 것 같습니다!